testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -104,3 +104,20 @@ Feature: Create activation keys
     And I enter "PROXY-KEY-x86_64" as "key"
     And I click on "Create Activation Key"
     Then I should see a "Activation key Proxy Key x86_64 has been created" text
+
+  Scenario: Create an activation key for the build host
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I wait until I do not see "Loading..." text
```

Comment:
please change this to `And I wait until child channels appear` or whatever new step i have introduced for activation keys

Reply:
Fixed.